src/videoosd.html

```diff
@@ -31,7 +31,7 @@ <h3 class="osdTitle"></h3>
                 <div class="osdTextContainer endTimeText" style="margin: 0 0 0 .25em;"></div>
             </div>
 
-            <div class="buttons">
+            <div class="buttons focuscontainer-x">
```

Comment:
what does `focuscontainer-x` mean?

Reply:
User navigates horizontally in such container. On Up/Down Arrow button, focus moves out from container.

Reply:
is this class already defined somewhere?

Reply:
<details>
  <summary>focusManager</summary>

  https://github.com/jellyfin/jellyfin-web/blob/17ea143b6b32b2d8de6d4ed6e77988017f3df71e/src/components/focusManager.js#L165-L208
</details>